units/initrd-battery-check.service.in

```diff
@@ -12,10 +12,14 @@ Description=Check battery level during early boot
 Documentation=man:initrd-battery-check.service(8)
 DefaultDependencies=no
 AssertPathExists=/etc/initrd-release
-Before=local-fs-pre.target
+After=plymouth-start.service
+Before=local-fs-pre.target local-fs.target
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-battery-check
 FailureAction=poweroff-force
+
+[Install]
+WantedBy=sysinit.target
```

Comment:
hmm, i don't think this makes sense. the initrd is not where people call "systemctl enable"...

THis should be hooked in statically, and probably from initrd.target and not sysinit.target so that people can use the same systemd.rpm both as initrd and as regular OS without failure.